js/packages/teams-ai/src/augmentations/Augmentation.ts

```diff
@@ -29,7 +34,7 @@ export interface Augmentation<TValue = any> extends PromptResponseValidator<TVal
     createPlanFromResponse(
         context: TurnContext,
         memory: Memory,
-        response: PromptResponse<TValue>
+        response: PromptResponse<TContent>
```

Comment:
DefaultResponseValidator TValue was switched to TContent. What is the intended difference between the two?

Reply:
![image](https://github.com/microsoft/teams-ai/assets/14900841/6d9324ca-6ebf-4f59-9fb6-2eca1212ae02)
![image](https://github.com/microsoft/teams-ai/assets/14900841/0242484b-dc47-4528-a3b9-c212ec605ea3)